config/api/assessment_schema.json

```diff
@@ -0,0 +1,382 @@
+{
+  "id": "#{Rails.root}/config/api/assessment_schema.json",
```

Comment:
I think this should be `"$id"`. See http://json-schema.org/understanding-json-schema/structuring.html#the-id-property

Reply:
The gem uses draft 4, where it's `id` not `$id`